commit c6293098c5538ff9ba7272b61eeb7410fb22462b
Author: Hubert Figuière <hub@figuiere.net>
Date:   Wed May 2 00:03:21 2018 -0400

    cargo-patch

diff --git a/mp4parse/Cargo.toml b/mp4parse/Cargo.toml
index ad74383..c62182f 100644
--- a/mp4parse/Cargo.toml
+++ b/mp4parse/Cargo.toml
@@ -19,13 +19,9 @@ exclude = [
   "*.mp4",
 ]
 
-[badges]
-travis-ci = { repository = "https://github.com/mozilla/mp4parse-rust" }
 
 [dependencies]
 byteorder = "1.2.1"
-afl = { version = "0.3", optional = true }
-abort_on_panic = { version = "1.0.0", optional = true }
 bitreader = { version = "0.3.0" }
 num-traits = "0.2.0"
 mp4parse_fallible = { version = "0.0.1", optional = true }
@@ -35,5 +31,4 @@ log = "0.4"
 test-assembler = "0.1.2"
 
 [features]
-fuzz = ["afl", "abort_on_panic"]
-craw = []
\ No newline at end of file
+craw = []
diff --git a/mp4parse_capi/Cargo.toml b/mp4parse_capi/Cargo.toml
index b1e2690..ec37951 100644
--- a/mp4parse_capi/Cargo.toml
+++ b/mp4parse_capi/Cargo.toml
@@ -18,8 +18,7 @@ exclude = [
   "*.mp4",
 ]
 
-[badges]
-travis-ci = { repository = "https://github.com/mozilla/mp4parse-rust" }
+build = false
 
 [dependencies]
 byteorder = "1.2.1"
@@ -33,9 +32,5 @@ num-traits = "0.2.0"
 [dev-dependencies]
 env_logger = "0.5.3"
 
-[build-dependencies]
-cbindgen = "0.5.2"
-
 [features]
-fuzz = ["mp4parse/fuzz"]
-craw = ["mp4parse/craw"]
\ No newline at end of file
+craw = ["mp4parse/craw"]
